rasa/cli/evaluate.py

```diff
@@ -147,9 +147,14 @@ def _run_markers(
             "Please see errors listed above and fix before running again."
         )
 
+    # Calculate telemetry
     # Subtract one to remove the virtual OR over all markers
     num_markers = len(markers) - 1
-    telemetry.track_markers_parsed_count(num_markers)
+    max_depth = markers.depth() - 1
+    # Find maximum branching of marker
+    branching_factor = max(len(marker) - 1 for marker in markers)
```

Comment:
`branching_factor = max(len(sub_marker) - 1  for marker in markers.sub_markers for for sub_marker in marker)` to exclude the artificial Or marker?

Reply:
Oh yep, otherwise we can end up with cases where the number of markers (which we already get) is returned twice - good spot